src/Dialog/index.js

```diff
@@ -100,7 +100,10 @@ const Dialog = ({
             </div>
             <div
               ref={contentRef}
-              className="nds-dialog-content nds-typography padding--top--xs"
+              className={cc([
+                "nds-dialog-content nds-typography padding--top--xs",
+                { "padding--bottom--l": !footer },
```

Comment:
```suggestion
                { "padding--bottom--xl": !footer },
```

Should it be xl to match what we have on the footer? https://github.com/narmi/design_system/blob/29a3a61b0097a7c59dbc0e46bd0e302ad35a3c1b/src/Dialog/index.scss#L101

Otherwise LGTM